docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+- The *tenure* area, which has 5% of its total memory available as free memory. All of this free memory is in the long object allocation area(LOA). No free memory is available in the short object allocation area(SOA).
+
+The `<gc-op>` elements and their child elements contain information about the increment’s operations and sub-operations. The final increment of the `gencon` global cycle consists of multiple operations, each logged with a `<gc-op>` element. The type of operation is shown by the `<gc-op>` `type` attribute. There are 5 types of operation involved in the final increment of the example log:
+
+1. `Rs-scan`
+2. `Card-cleaning`
+3. `Mark`
+4. `Classunload`
+5. `Sweep`
+
+**Note:** The final increment of a `gencon` global cycle can include an optional compact sub-operation. 
+
+**Note:** For more information about the different types of GC operation, see [GC Operations](gc_overview.md#gc-operations).
+
+```xml
+<gc-op id="12382" type="rs-scan" timems="3.525" contextid="12364" timestamp="2020-10-18T13:35:44.598">
+
+<scan objectsFound="11895" bytesTraced="5537600" workStackOverflowCount="0" />
+
+</gc-op>
+
+<gc-op id="12383" type="card-cleaning" timems="2.910" contextid="12364" timestamp="2020-10-18T13:35:44.601">
+
+<card-cleaning cardsCleaned="3603" bytesTraced="5808348" workStackOverflowCount="0" />
+
+</gc-op>
+
+<gc-op id="12384" type="mark" timems="6.495" contextid="12364" timestamp="2020-10-18T13:35:44.607">
+
+<trace-info objectcount="1936" scancount="1698" scanbytes="61200" />
+
+<finalization candidates="389" enqueued="1" />
+
+<ownableSynchronizers candidates="5076" cleared="523" />
+
+<references type="soft" candidates="18420" cleared="0" enqueued="0" dynamicThreshold="32" maxThreshold="32" />
+
+<references type="weak" candidates="19920" cleared="114" enqueued="60" />
+
+<references type="phantom" candidates="671" cleared="50" enqueued="50" />
+
+<stringconstants candidates="40956" cleared="109" />
+
+<object-monitors candidates="182" cleared="51" />
+
+</gc-op>
+
+<gc-op id="12385" type="classunload" timems="1.607" contextid="12364" timestamp="2020-10-18T13:35:44.609">
+
+<classunload-info classloadercandidates="425" classloadersunloaded="6" classesunloaded="2" anonymousclassesunloaded="1" quiescems="0.000" setupms="1.581" scanms="0.019" postms="0.007" />
+
+</gc-op>
+
+<gc-op id="12386" type="sweep" timems="9.464" contextid="12364" timestamp="2020-10-18T13:35:44.618" />
+```
+
+The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory in the heap, similar to `<gc-start>`. In the example, 60% of the heap now contains free memory as a result of the final global cycle increment. The *nursery* area remains unchanged which means the final increment reclaimed memory only in the *tenure* space, which now has 51% of free memory available. The LOA is now all available as free memory. The SOA has 48% of it's memory available as free memory.
```

Comment:
In general case, since this is a global cycle, memory will be reclaimed in both nursery and tenure. However, in this particular case nursery was already very free and reclamation was very low. If you carefully see free memory in bytes  jumped from 234609440 to 235516088. Try to find a better example, but if not, at least you  should update the wording.

Reply:
**Are these paragraphs correct for the following log output?**

```xml
<gc-start id="12380" type="global" contextid="12364" timestamp="2020-10-18T13:35:44.594">

<mem-info id="12381" free="277048640" total="1073741824" percent="25">

<mem type="nursery" free="234609440" total="268435456" percent="87">

<mem type="allocate" free="234609440" total="241565696" percent="97" />

<mem type="survivor" free="0" total="26869760" percent="0" />

</mem>

<mem type="tenure" free="42439200" total="805306368" percent="5">

<mem type="soa" free="2173472" total="765040640" percent="0" />

<mem type="loa" free="40265728" total="40265728" percent="100" />

</mem>
```
For this example, at the start of this GC increment, the *tenure* area is low on free memory, as expected. 25% of the total heap is available as free memory, which is split between the following areas of the heap:

- The *nursery* area, which has 234.6MB (234,609,440MB) of free memory available. The free memory is only available in the *allocate* space of the *nursery* area.The *survivor* space of the *nursery* area is reported as 'full' to reflect that no available memory is available to allocate to the mutator threads. The entire *survivor* space is reserved for GC operations during the GC increment. 
- The *tenure* area, which has 42MB (42,439,200MB) available as free memory, which is only 5% of its total memory. Most of this free memory is in the large object allocation area(LOA). Almost no free memory is available in the small object allocation area(SOA). 

**and**

```xml
<gc-end id="12387" type="global" contextid="12364" durationms="24.220" usertimems="86.465" systemtimems="0.000" stalltimems="2.846" timestamp="2020-10-18T13:35:44.618" activeThreads="4">

<mem-info id="12388" free="650476504" total="1073741824" percent="60">

<mem type="nursery" free="235516088" total="268435456" percent="87">

<mem type="allocate" free="235516088" total="241565696" percent="97" />

<mem type="survivor" free="0" total="26869760" percent="0" />

</mem>

<mem type="tenure" free="414960416" total="805306368" percent="51" micro-fragmented="98245682" macro-fragmented="0">

<mem type="soa" free="374694688" total="765040640" percent="48" />

<mem type="loa" free="40265728" total="40265728" percent="100" />

</mem>
```
60% of the heap now contains free memory as a result of the final global cycle increment, which is split between the following areas of the heap:

- The *nursery* area, which has gained 0.91MB of of free memory (the *nursery area now has 235.2MB (235,516,088MB) available as free memory. At the start of the final increment the area had 234.6MB (234,609,440MB) of free memory available).
- The *tenure* area, which has gained 372MB (372,521,216MB) of free memory. (the *tenure* area now has 414MB (414,960,416MB) available as free memory. At the start of the final increment area had 42MB (42,439,200MB) of free memory available.

**Note:** The global GC cycle runs to reclaim memory in the *tenure* area. The freeing up of memory in the *nursery* area is achieved by using the partial GC cycle. For more information, see [`gencon` Policy(default)](gc.md#gencon-policy-(default)).

**QUESTION** does the global GC cycle final increment reclaim memory from the nursery as well as the tenure area? - I thought that the reclaim reported in the nursery area was from the partial gc cycles, but the recorded. reclaim is between the start and end of the final increment only. If so, remove the **note** as it is incorrect.